src/components/plots/plot.ts

```diff
@@ -459,19 +459,21 @@ module Plottable {
       var entities: Plots.Entity[] = [];
       datasets.forEach((dataset) => {
         var drawer = this._datasetToDrawer.get(dataset);
-        dataset.data().forEach((datum: any, index: number) => {
-          var position = this._pixelPoint(datum, index, dataset);
+        var loopIndex = 0;
```

Comment:
the var name is misleading. `drawerIndex`, `entitiesIndex` `selectionIndex`, `drawerSelectionIndex` could be more appropriate